views/index.pug

```diff
@@ -1,21 +1,68 @@
 html
-  include partials/head
+head
+    link(rel="stylesheet" href="/css/styles.css")
 
-  body
-    h1= schittscreek.name
+    body
+     h1.title Welcome to Schitt's Creek!
+      h2 Select a Method:
+      li.methodsInfo 
+        p 
+          ul.methods GET/allCharacters
+        div#getAll
+            p Method: GET 
+            p Route: http://localhost:4053/characters/allCharacters 
```

Comment:
I would expect `http://localhost:4053/characters` to return all characters, the `allCharacters` seems redundant